docs/data/charts/line-demo/LiveLineChartNoSnap.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { LineChart } from '@mui/x-charts/LineChart';
+
+const dateFormatter = Intl.DateTimeFormat(undefined, {
+  month: '2-digit',
+  day: '2-digit',
+});
+const oneDay = 24 * 60 * 60 * 1000; // in milliseconds
+
+const length = 16;
+const firstDataSeed = randBetween(0, 5000);
+const secondDataSeed = randBetween(0, 5000);
+const initialFirstData = Array.from<number>({ length }).map(
+  (_, __, array) => (array.at(-1) ?? firstDataSeed) + randBetween(-500, 500),
```

Comment:
`Array.at` is supported across our [supported platforms](https://next.mui.com/material-ui/getting-started/supported-platforms/) ([source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at#browser_compatibility))

Reply:
This is the demo, it doesn't really matter I suppose